src/generate_pisa_index.cpp

```diff
@@ -155,7 +133,6 @@ int main(int argc, char const *argv[])
     google::protobuf::io::ZeroCopyInputStream* postings_stream = new google::protobuf::io::IstreamInputStream(&postings_data);
     google::protobuf::io::CodedInputStream coded_stream(postings_stream);
 
-    uint32_t term_id = 0;
     while (true) {
         uint32_t message_size;
         if (!coded_stream.ReadVarint32(&message_size)) {
```

Comment:
What do you think of:
```cpp
uint32_t message;
while (coded_stream.ReadVarint32(&message_size)) {
...
}

Btw, is this the usual way you handle the end of input in protobuf? Or is it something about our format? Asking because of the "assuming" in the comment, as if it's something specific to our format...
```

Reply:
Looks good. Yeah, I believe what you did is correct here.